astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -143,33 +143,25 @@ def test_names():
 
 @pytest.mark.remote_data
 def test_name_resolve_cache(tmpdir):
```

Comment:
NOTE: This test was not added until #9162 and only available from 4.1 onwards, so this won't be backported to 4.0.x. This is just a test, so no functionality change.